cmd/common_konnect.go

```diff
@@ -29,13 +29,18 @@ var addresses = []string{
 	defaultLegacyKonnectURL,
 }
 
-func authenticate(ctx context.Context, client *konnect.Client, host string) (konnect.AuthResponse, error) {
+var konnectAuthResponse konnect.AuthResponse
```

Comment:
Yikes. This global variable seems very wrong.

Can you please instead change the return value for LoginV2 to be a custom struct that is populated no matter what authentication method is used?
And that can be populated using me API of Konnect.

Am I missing something?